receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -163,52 +179,117 @@ func (k *K8sAPIServer) GetMetrics() []pmetric.Metrics {
 		attributes["NodeName"] = k.nodeName
 	}
 	attributes[ci.SourcesKey] = "[\"apiserver\"]"
-	md := ci.ConvertToOTLPMetrics(fields, attributes, k.logger)
-	result = append(result, md)
+	return ci.ConvertToOTLPMetrics(fields, attributes, k.logger)
+}
 
-	for service, podNum := range k.epClient.ServiceToPodNum() {
+func (k *K8sAPIServer) getNamespaceMetrics(clusterName, timestampNs string) []pmetric.Metrics {
+	var metrics []pmetric.Metrics
+	for namespace, podNum := range k.podClient.NamespaceToRunningPodNum() {
 		fields := map[string]interface{}{
-			"service_number_of_running_pods": podNum,
+			"namespace_number_of_running_pods": podNum,
```

Comment:
has this metric always been incorrect? Is this a breaking change for CI?

Reply:
No this is just git diffs looking weird since i extracted this into diff functions.
This particular metric is still available [here](https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/20/files/81d5295bd7c6e299155a5b62db05b65adf748282#diff-1a583ee28f5c5b578fb9623462476ff5b07e8f862be6465571fbf4a7c731e9d1R273)